app/views/posts/index.html.erb

```diff
@@ -1 +1,44 @@
-<h1>Here is a list of posts for a given user</h1>
+<div class="wrapper">
+  <div class="userNameWrapper">
+          <div><img alt="Profile P."/></div>
+          <div class="userName">
+              <h2> 
+                  Name: <%= @user.name %>
+              </h2>
+              <h6>Number of posts: <%= @user.posts.count%></h6>
+          </div>
+      </div>
+
+  <% @posts.each_with_index do |post, index| %>
+
+    <div class="post">
+      <%= link_to user_post_path(@user, post), class: 'post-link' do %>
+      
+        <h2>Post #<%= index+1 %></h2>
+        <h3><%= post.title %></h3>
```

Comment:
- [ ] I think here when I try to create a new post in the console when I try to insert a new post it shows me this message error that disables me from seeing the posts page(index) and (show)page so kindly fix the issue displayed here to make your project work as expected 🤩🚀🔥

- No posts here!
![image](https://github.com/Jrmcuervo/Blog-app_ror/assets/58553711/52d39c6d-e07c-4f4b-8288-16ea4d980923)


- The Error message ⬇
![image](https://github.com/Jrmcuervo/Blog-app_ror/assets/58553711/cc179672-637e-454c-9777-b5135a9ce71b)